sql.html.markdown

```diff
@@ -0,0 +1,106 @@
+---
+language: SQL
+filename: learnsql.sql
+contributors:
+  - ["Bob DuCharme", "http://bobdc.com/"]
+https://github.com/adambard/learnxinyminutes-docs
```

Comment:
@bobdc This line with the link to this repo is breaking the build. Please remove this.

Reply:
Line 6

Reply:
Sorry, that must have been an accidental paste. Removed.